Resources/views/Block/ajax.html.twig

```diff
@@ -0,0 +1,18 @@
+{% block block %}
+    <div class="{{ block.settings.class }}" id="block-{{ block.id }}" >
+        <div class="info-box">
+            <span class="info-box-icon bg-{{ block.settings.color }}"><i class="{{ block.settings.icon }}"></i></span>
+            <div class="info-box-content">
+                <span class="info-box-text">{{ block.settings.title }}</span>
+                <span class="info-box-number"></span>
+            </div>
+        </div>
+    </div>
+    <script>
+        $.ajax({
+            url: "{{ block.settings.url }}"
+        }).done(function( data ) {
+            $('#block-{{ block.id }} .info-box-number').html(data);
+        });
+    </script>
```

Comment:
Javascript inside a twig file is maybe not the best way to handle this. For the moment, this looks simple, but if 401, 500 status code handling is added, this could get complicated. Please try to have no js at all here if you can cc @ju1ius